src/redux/configureStore.js

```diff
@@ -0,0 +1,8 @@
+import { createStore, combineReducers } from 'redux';
+import * as Books from './books/books';
+import * as Categories from './categories/categories';
+
+const rootReducer = combineReducers(Books.default, Categories.default);
+const store = createStore(rootReducer);
+
+export default store;
```

Comment:
- [ ] I would recommend you fix this warning in the terminal (see image). Based on your configuration, React responds that you do not have a valid reducer. Kindly fix that to avoid future issues.

  ![fkfk](https://user-images.githubusercontent.com/36542986/160891364-1bfab0b1-3ea7-456c-8cd8-31bde40816ef.jpg)

  **SUGGESTION:** I think this error has to do with your imports and `rootReducer` setup cause when I did something like so, it resolved the issue.
  ```
  import Books from './books/books';   // default import
  import Categories from './categories/categories';   // default import
  
  const rootReducer = combineReducers({ Books, Categories });   // notice curly braces
  ```

Reply:
Thank you so much, I did work.